uint8 count
geometry_msgs/Point puck_1
geometry_msgs/Point puck_2
geometry_msgs/Point puck_3
float64 dist_12
float64 dist_13
float64 dist_23
